skimage/metrics/simple_metrics.py

```diff
@@ -259,3 +263,51 @@ def normalized_mutual_information(image0, image1, *, bins=100):
     H01 = entropy(np.reshape(hist, -1))
 
     return (H0 + H1) / H01
+
+
+def enhancement_measure(image, size=3, eps=1e-6):
```

Comment:
Is there some reasoning between choosing `eps=1e-6` as the default? Is something else more reasonable?